components/sections/Navbar.tsx

```diff
@@ -55,7 +55,7 @@ const Navbar = () => {
               </NextLink>
             ))}
 
-            <Link variant="navlink">Language</Link>
+            {/* <Link variant="navlink">Language</Link> */}
           </Stack>
```

Comment:
For opening in new-tab, we can add this configuration at link level.
```
const links = [
  {
    title: "Alphanet",
    link: "/alphanet",
    newTab: false,
  },
  {
    title: "Community",
    link: COMMUNITY_URL,
    newTab: false,
  },
  {
    title: "Litepaper",
    link: "/litepaper",
    newTab: true,
  },
  // commenting it for now until we have page for it
  // {
  //   title: "Learn",
  //   link: "#",
  // },
  {
    title: "Blog",
    link: BLOG_URL,
    newTab: false,
  },
];
```
And in Navbar link handling : 
 ```
{links?.map((link) => (
              <NextLink key={link.title} href={link.link} passHref>
                <Link
                  variant="navlink"
                  rel="noopener noreferrer"
                  target={link.newTab ? "_blank" : "_self"}
                >
                  {link.title}
                </Link>
              </NextLink>
            ))}
```